<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newnew</class>
 <widget class="QDialog" name="newnew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>430</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>593</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>511</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>4</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/Images/2D Graphic Label.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>191</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>169</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>222</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Help!</string>
   </property>
  </widget>
  <widget class="QLabel" name="falling">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>-1</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/Images/fallingleaves.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="userswitch">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>create user</string>
   </property>
  </widget>
  <widget class="QComboBox" name="userCombo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>69</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Guest</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Admin</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="adminCode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="codeLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ERRORADMINCODE">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>falling</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>loginButton</zorder>
  <zorder>nameEdit</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>passwordEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>createButton</zorder>
  <zorder>userswitch</zorder>
  <zorder>userCombo</zorder>
  <zorder>adminCode</zorder>
  <zorder>codeLabel</zorder>
  <zorder>ERRORADMINCODE</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>nameEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
